app/src/main/res/values/dimens.xml

```diff
@@ -1,4 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <dimen name="dp_4">4dp</dimen>
+    <dimen name="dp_48">48dp</dimen>
+    <dimen name="sp_11">11sp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.sp_11` appears to be unused
 Unused resources make applications larger and slow down builds.